src/v8/mui-x-codemods.ts

```diff
@@ -0,0 +1,5 @@
+import { executeCommand } from "../util/execute-command.util";
+
+export default async function executeMuiXCodemods() {
+    await executeCommand("npx", ["@mui/x-codemod@latest", "v6.0.0/data-grid/preset-safe", "admin/src"]);
```

Comment:
Shouldn't this include v7 codemods as well?

Reply:
Yes: 39d1d94b81429f15a1ed31203c7479efc735285c